### PR TITLE
BODP-5637: Update requires attention screen banner

### PR DESCRIPTION
**Pull request by marcusher-ito**
Completed UI changes, percentage calculation logic and unit tests to check percentage logic.

May require implementation of new logic from BODP-5634 (Seasonal Services & Staleness) to update new metrics, and small updates to UI.

### CODE REVIEW

**Comment by ITO-PaulHolland** [comment]
screenshot

**Comment by marcusher-ito** [comment]
> 

Thanks Paul - will add one when it's ready for review in case it changes!

**Comment by marcusher-ito** [comment]
Screenshot 2023-01-30 at 10 25 38
Screenshot 2023-01-30 at 10 26 08

**Comment by marcusher-ito** [comment]
- Added gov.uk tag for compliant/non-compliant tag
- Used updated staleness/seasonal services logic for new Services Requiring Attention metrics
- Updated unit tests

Screenshot 2023-02-15 at 14 09 54
Screenshot 2023-02-15 at 14 43 07
Screenshot 2023-02-15 at 14 44 31

**Comment by anguskbell** [comment]
I know it's been copied from the design, and it's not part of your ticket, but when there are no search results, it says:
* User another search term

when clearly it should say "Use".

Could you fix this?

**Comment by marcusher-ito** [comment]
> I know it's been copied from the design, and it's not part of your ticket, but when there are no search results, it says:
> 
> * User another search term
> 
> when clearly it should say "Use".
> 
> Could you fix this?

Well spotted, thanks Angus! That's now fixed:

Screenshot 2023-02-17 at 12 11 58

File: transit_odp/publish/templates/publish/requires_attention.html [hunk 1]
```diff
@@ -5,49 +5,80 @@
...
+                </div>
+            </div>
+            <div class="govuk-grid-column-one-quarter">
+                <div class="feed-stat">
+                    <span class="feed-stat__value">{{ services_require_attention_percentage }}%</span>
+                    <span class="feed-stat__label govuk-!-font-size-16 govuk-!-margin-bottom-2">
+                        {% trans 'Services requiring<br>attention' %}
+                    </span>
+                    {% if services_require_attention_percentage == 0 %}
+                        <span class="govuk-tag govuk-tag--green" style="letter-spacing: 0px;">{% trans 'Compliant' %}</span> {% else %}
```
Comment (anguskbell) [inline transit_odp/publish/templates/publish/requires_attention.html:70]: I would put the `else` on a separate line, to make it more readable.

File: transit_odp/publish/tests/test_views.py [hunk 2]
```diff
@@ -2045,3 +2052,71 @@ def test_require_attention_all_variations(
...
+    # One out of season seasonal service reduces in scope services to 8
+    assert response.context["services_require_attention_percentage"] == 75
+
+
+@pytest.mark.parametrize(
+    "pathname,pathargs,pathkwargs",
+    [
+        ("requires-attention", [], {"pk1": 1}),
+    ],
+)
```
Comment (anguskbell) [inline transit_odp/publish/tests/test_views.py:2064]: You don't need to parameterise it if you're only going to run it with one set of input data.

**Review by anguskbell — APPROVED** [review]

**Pull request merged**